ortho_seq_code/orthogonal_polynomial.py

```diff
@@ -35,33 +22,53 @@ def orthogonal_polynomial(
     out_dir,
     alphbt_input,
     min_pct,
+    pheno_name,
 ):
 
     """Program to compute orthogonal polynomials up to 2nd order"""
     start_time = time.time()
     out_dir = create_dir_if_not_exists(out_dir)
     global i
-    with open(filename) as f:
-        seq = f.readlines()
+    print("")
+    if pheno_file != None:
+        with open(filename) as f:
+            seq = f.readlines()
+        with open(pheno_file) as f2:
+            phenotype = f2.readlines()
+        naming_phenotype = os.path.basename(f2.name)
+        onefile = False
+    else:
+        if os.path.splitext(filename)[1] == ".xlsx":
+            df = pd.read_excel(filename, engine="openpyxl", header=None)
+        else:
+            df = pd.read_csv(filename, header=None)
+        phenotype = df[1]
+        f = filename
+        naming_phenotype = os.path.splitext(os.path.basename(filename))[0]
+        onefile = True
```

Comment:
rename onefile to pheno_seqs_same_file or something of the sort. It is helpful if you are writing an if loop later on the variable if pheno_seqs_same_file makes it easy to read and understand